### PR TITLE
Update docs to reflect updated 7 day period length

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [X] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [ ] 2
- [X] 3

**What this does:**
Documents the change of period length from 24 hours to 7 days (168 hours).

**Why it's needed:**
We need to properly document the changes to period length

### CODE REVIEW

**Comment by derekpierre** [comment]
Need to update some sample CLI output ... 🤯

File: docs/source/glossary.rst
Comment (arjunhassard) [inline docs/source/glossary.rst:51]:
```suggestion
      A timeframe of approximately 7 days in the NuCypher Network. This parameter is used as a minimum unit for policy duration, and is also the cadence with which workers must make an on-chain commitment to being online and available. Note that period durations were ~24h in the genesis (previous) protocol and could change again in the future.
```

File: docs/source/staking/overview.rst
Comment (arjunhassard) [inline docs/source/staking/overview.rst:31]:
```suggestion
a commitment transaction before the start of each period (every 7 days), which costs approximately 200k gas.
```

File: docs/source/staking/stake_initialization.rst
Comment (arjunhassard) [inline docs/source/staking/stake_initialization.rst:37]:
```suggestion
    periodic automated commitments (every 7 days) which cost at least ~200k gas, depending on how many sub-stakes you have. Be sure to consider this
```

File: docs/source/staking/stake_initialization.rst
Comment (arjunhassard) [inline docs/source/staking/stake_initialization.rst:231]: This is a bit confusing. I guess until there's < 4 weeks to go, it will still print 5 periods remaining. Perhaps to show this better we could set the illustrative Termination date as July 23rd (i.e. 4 weeks and 6 days)

**Review by arjunhassard — COMMENTED** [review]
Nice! We probably need a whole docs page on how to migrate to the 7 day protocol, especially for those _laggards_

File: docs/source/staking/stake_initialization.rst
Comment (derekpierre) [reply docs/source/staking/stake_initialization.rst:231]: Of note is that I manually edited these outputs.

The remaining periods is calculated as follows:

```python
    @property
    def periods_remaining(self) -> int:
        """Returns the number of periods remaining in the stake from now."""
        current_period = self.staking_agent.get_current_period()
        return self.final_locked_period - current_period + 1
```

Technically the value could be anything <= 5. However, since this is a new stake, the current date is Jun 19th, 2020. You'll notice that the old value was for 30 periods (30 days), but remaining was 31 because of the same calculation;  because the first commitment is for the next period. Hoping that @vzotova can confirm whether this edit is correct...

I can modify the termination date if that is too confusing, but it was essentially the same thing we had before.

File: docs/source/glossary.rst
Comment (derekpierre) [reply docs/source/glossary.rst]: ✔️

File: docs/source/staking/overview.rst
Comment (derekpierre) [reply docs/source/staking/overview.rst]: ✔️

File: docs/source/staking/stake_initialization.rst
Comment (derekpierre) [reply docs/source/staking/stake_initialization.rst]: ✔️

**Comment by derekpierre** [comment]
> Nice! We probably need a whole docs page on how to migrate to the 7 day protocol, especially for those laggards

Absolutely - that is on my list.

File: docs/source/support/faq.rst
Comment (vzotova) [inline docs/source/support/faq.rst:32]: I guess better to add which day midnight

File: docs/source/staking/stake_initialization.rst
Comment (vzotova) [reply docs/source/staking/stake_initialization.rst:231]: looks correct, ideally to try it on lynx

File: docs/source/support/faq.rst
Comment (derekpierre) [reply docs/source/support/faq.rst:32]: A specific day would be great. Presumably this depends on when we upgrade the contract; is there a plan to do the upgrade on a specific day?

Comment (vzotova) [reply docs/source/support/faq.rst:32]: I think it should be unrelated to upgrade day, it's day when current period division by 7 without a remainder

File: docs/source/staking/stake_initialization.rst
Comment (derekpierre) [reply docs/source/staking/stake_initialization.rst]: Added CLI output from lynx operations.

**Comment by derekpierre** [comment]
Provide `[some date]` value in `period_duration.rst` for the date when the DAO propsal is possibly enacted, before merging.

**Comment by derekpierre** [comment]
In the interim, once we get approvals, we can merge in current state, and the create a new WIP PR to modify the `[some date]` placeholder.

File: docs/source/architecture/period_duration.rst
Comment (KPrasch) [inline docs/source/architecture/period_duration.rst:16]:
```suggestion
The cost of the commitment transaction is approximately 200k gas units but may be higher depending on the number of
sub-stakes a staker has. Given the significant increase in Ethereum gas price since network launch, the cost of the daily
 ```

File: docs/source/architecture/period_duration.rst
Comment (KPrasch) [inline docs/source/architecture/period_duration.rst:21]: This is not exactly accurate since transactions still cost the same amount, there are just fewer of them required.

```suggestion
 Reducing the total amount of gas fees required to run a worker became a necessity, and there have been multiple mitigations:
```

File: docs/source/architecture/period_duration.rst
Comment (KPrasch) [inline docs/source/architecture/period_duration.rst:17]:
```suggestion
commitment transaction significantly increased and became untenable. Particularly for short stake duration and value closer to the minimum
```

File: docs/source/architecture/period_duration.rst
Comment (KPrasch) [inline docs/source/architecture/period_duration.rst:32]:
```suggestion
Taking a long-term perspective, lowering Worker overhead encourages more participation, and enables more
```

File: docs/source/architecture/period_duration.rst
Comment (KPrasch) [inline docs/source/architecture/period_duration.rst:52]:
```suggestion
will also be minted once every 7 days (instead of daily), however the reward rate remains the same (the minted value will be 7 days worth).
```

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/architecture/period_duration.rst
Comment (derekpierre) [reply docs/source/architecture/period_duration.rst]: ✔️

File: docs/source/staking/stake_initialization.rst
Comment (cygnusv) [inline docs/source/staking/stake_initialization.rst:258]: Wow, we probably should change this parameter to 1 period.

**Review by cygnusv — APPROVED** [review]

File: docs/source/staking/stake_initialization.rst
Comment (derekpierre) [reply docs/source/staking/stake_initialization.rst:258]: Unfortunately, I am not sure that it can be lower than 2 periods. If I remember correctly, 2 periods might the minimum to prevent workers from circumventing slashing... @vzotova ?

**Review by KPrasch — APPROVED** [review]
Thanks @derekpierre !

**Pull request merged**
